subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -99,7 +99,7 @@ dependencies {
         api(libs.jgit)                  { version { strictly("5.7.0.202003110725-r"); because("Upgrade has breaking API changes") }}
         api(libs.joda)                  { version { strictly("2.10.4") }}
         api(libs.joptSimple)            { version { strictly("5.0.4"); because("needed to create profiler in Gradle profiler API") }}
-        api(libs.jsch)                  { version { strictly("0.1.55") }}
+        api(libs.jsch)                  { version { strictly("0.1.70") }; because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement")}
```

Comment:
Should it not be:
```
api(libs.jsch) { version { strictly("0.1.70"); because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement") }}
```

?

Reply:
Sorry, I don't understand, what do you mean?

Reply:
Hi. Sorry, I'll try to clarify...

It was previously

```
api(libs.jsch)                  { version { strictly("0.1.55") }}
```

You changed to:

```
api(libs.jsch)                  { version { strictly("0.1.70") }; because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement")}
```

However, by looking at the line above (line 101, https://github.com/gradle/gradle/pull/19060/files/c2e83002f3d9987d5d5ffcb49a01716c113b4894#diff-961415f8ddcdd04d65d758b9f9a23809f36a53eb936e7e062129c74f1c23d61cR101) it looks like `;` should be directly after `strictly("0.1.70")` and that `) }}` should be placed at the end of the line. 

I.e., 
```
api(libs.jsch) { version { strictly("0.1.70"); because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement") }}
```

Reply:
Ah right, thanks. for explaining further. This is probably more clear if it's formatted differently

```kotlin
// code in the PR:
api(libs.jsch) {
    version { strictly("0.1.70") }
    because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement")
}

// your proposal:
api(libs.jsch) {
    version {
        strictly("0.1.70")
        because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement")
    }
}
```

`because()` can be used to document the dependency, or the version, or both. In this case there's nothing special about the version, it's just the latest version. I do want to document the dependency though, as it's a slightly unusual case.

Reply:
Ok, thanks for the explanation.